src/mpi/coll/cvars.txt

```diff
@@ -170,6 +170,17 @@ cvars:
         nb                      - Force nonblocking algorithm
         remote_send_local_bcast - Force remote-send-local-bcast algorithm
 
+    - name        : MPIR_CVAR_IBCAST_ERROR_CHECKING
```

Comment:
This cvar is neither general nor specific. It doesn't apply to all IBCAST algorithms, so it is a wrong name if view it as specific. It singles out ibcast yet I don't see the rationale for singling out ibcast -- usually when user desire error checking, they desire it for all collectives.

How about we just use the macro `HAVE_ERROR_CHECKING`?

Reply:
I think we had a debate on that a long time back. I will have to recall the reason for this. I think It has something to with the MPI standard, since MPI runtime is required to report error in some cases in ibcast. So we prefer not to eliminate it by using the macro. With the specific env var,  the user basically specify he/she is ok with the non-standard behavior for performance reason.

Reply:
You were right. I found we have a later commit actually removed the CVAR. I squashed it to this commit now, so it is gone,